client/src/templates/layouts/NavigationLayout.js

```diff
@@ -310,7 +310,6 @@ const NavigationLayout = (props) => {
       });
     }
   };
-
   const renderThanksModal = () => (
```

Comment:
add the line space back, between 312 and 313.

Reply:
Thats odd! Auto prettier deleted it but It makes sense to have it. Thank you for pointing this out.